libs/client-api/src/http_publish.rs

```diff
@@ -292,15 +292,15 @@ impl Client {
     publish_name: &str,
   ) -> Result<T, AppResponseError>
   where
-    T: serde::de::DeserializeOwned,
+    T: serde::de::DeserializeOwned + 'static,
   {
     tracing::debug!(
       "get_published_collab: {} {}",
       publish_namespace,
       publish_name
     );
     let url = format!(
-      "{}/api/workspace/published/{}/{}",
+      "{}/v1/api/workspace/published/{}/{}",
```

Comment:
As we are using /api for our nginx config for calls to appflowy cloud, we probably should still keep the `api` prefix instead.

Reply:
`v1` as what you mentioned can potentially conflict with namespace. This also follows the convention of `v1` the previous endpoints.

Reply:
oh sorry, I was mistaken, you were right. I've made the changes